src/Mover.php

```diff
@@ -77,16 +78,39 @@ public function movePackage(Package $package)
                         $contents = $replacer->replace($contents);
 
                         if ( is_a($replacer, ClassmapReplacer::class)) {
-                            $replacedClasses = array_merge($replacedClasses, $replacer->replacedClasses);
+                            $this->replacedClasses = array_merge($this->replacedClasses, $replacer->replacedClasses);
                         }
 
                         $filesystem->put($targetFile, $contents);
                     }
                 }
             }
         }
+    }
+
+    public function replaceClassmapNames()
+    {
+        $classmap_path = $this->workingDir . $this->config->classmap_directory;
+        $finder = new Finder();
+        $finder->files()->in($classmap_path);
 
-        // Replace all replaced class names throughout the classmap dependencies here
+        $filesystem = new Filesystem(new Local($this->workingDir));
 
+        foreach ($finder as $file) {
+            $file_path = str_replace($this->workingDir, '', $file->getRealPath());
+            $contents = $filesystem->read($file_path);
+
+            foreach( $this->replacedClasses as $original => $replacement ) {
+                $contents = preg_replace_callback(
+                    '/\W('.$original.')(?:\(|\:\:)/U',
```

Comment:
This will miss a couple of scenarios.

The ones I can think of right now:

1. type-hints: `public function __construct( MyClass $obj );`
2. `new` without arguments: `$obj = new MyClass;`
3. class in string: `$class = 'MyClass'; $obj = new $class();`

It will also probably break when using relative namespaces, like the following:
```
namespace Vendor;
throw new Exception\MyException(); // FQCN: \Vendor\Exception\MyException
```

I'm not really sure how you can add cases like these without getting false positives as well, though. I don't think this can properly be handled by regexes, you should look into using `nikic/PHP-Parser` perhaps.